LondonBritishLibrary/orient/BLorient767.xml

```diff
@@ -52,48 +82,98 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                        <width>216</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>10.125</height>
+                                        <width>8.5</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="18" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is fairly written of the later half of  <date>18th century.</date>
+                                <desc>The manuscript is fairly written of the later half of <date>18th century.</date>
                                 </desc>
-                                
+
                                 <date notBefore="1750" notAfter="1800" resp="PRS10171Wright"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+
+                  <item xml:id="a1">
+                              <locus target="1r"/>
+                              <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                              <q xml:lang="gez">ዜና፡ አበው፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                          </q>
+                  </item>
+
+                  <item xml:id="a2">
+                  <locus target="#135r"/>
+                  <desc type="MixedNote">A list of the abbots of <placeName ref="INS0341DL">Dabra Libānos</placeName>
+                  from <persName ref="PRS3742ElsaE">ʾElsā</persName> to <persName ref="PRS11903Zakrestos">Za-Krǝstos</persName>.</desc>
+                  <q xml:lang="gez">ዘነበሩ፡ ዲበ፡ መንበረ፡ አቡነ፡ <space/> ዝውእቱ፡ አስማቲሆሙ፡ ኤልሳዕ፡ ፊልጶስ። ሕዝቅያስ፡ ቴዎድሮስ፡ ዮሐንስ።
+                ዮሐንስ፡ ካማ፡ እንድርያስ፡ መርሐ፡ ክርስቶስ፡ ጴጥሮስ፡ ያዕቆብ፡ ማትያስ፡ ዮሐንስ፡ ዕንባቆም፡ ዮሐንስ፡ ዘትረ፡ ወንጌል፡ አብርሃም፡ ዘወንጌል፡ ዘክርስቶስ፡ በ<space/></q>
+                  </item>
+
+                  <item xml:id="a3">
+                  <locus from="135r" to="136v"/>
+                  <desc type="GuestText"><title ref="LIT4336Ledata"/></desc>
+                  <q xml:lang="gez"/>
+                  <title xml:lang="gez" type="complete" ref="LIT4336Ledata"/>
```

Comment:
this second title is unnecessary, you can just add the type to the first one

Reply:
Sure, more economic that way.